tests/callbacks/test_lr_logger.py

```diff
@@ -50,14 +50,15 @@ def test_lr_logger_no_lr(tmpdir):
         assert result
 
 
-def test_lr_logger_multi_lrs(tmpdir):
+@pytest.mark.parametrize("logging_interval", ['step', 'epoch'])
+def test_lr_logger_multi_lrs(tmpdir, logging_interval):
```

Comment:
is `logging_interval=None` tested elsewhere?

Reply:
not yet, if `logging_interval=None`, then the assertion `assert all(len(lr) == expected_number_logged for k, lr in lr_logger.lrs.items())` will depend on individual scheduler